### PR TITLE
add UI structure and tailwind css

### PR DESCRIPTION
**Pull request by ReAliens**
in this pull request, I implemented the following:-
- add tailwind CSS 
- add prop-types package
- implement basic UI Structure

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,30 @@
...
+        </div>
+        <div className="flex flex-col w-1/5">
+          {signs.basic.map((sign) => (
+            <SignButton key={sign.id} sign={sign.name} type="basic" />
+          ))}
+        </div>
+      </div>
     </div>
   );
 }
```
Comment (rukundoeric) [inline src/App.js]: - [ ] <b>REQUIRED:</b> All of those components your imported here are supposed to be imported and used in `Calculator` component instead. and then use `Calculator` component in here. Kindly do that to comply with project requirements.

File: src/App.js [hunk 2]
```diff
@@ -1,25 +1,30 @@
-import logo from './logo.svg';
 import './App.css';
+import SignButton from './components/Buttons/BasicSigns';
+import NumberButton from './components/Buttons/NumberButton';
+import CalcInput from './components/CalcInput';
+import { numbers, signs } from './static/button';
 
 function App() {
   return (
```
Comment (rukundoeric) [inline src/App.js]: - [ ] <b>REQUIRED:</b> The `App.js` component should be class-based. Kindly fix that.

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @ReAliens, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 

## Required Changes ♻️
  - [ ] You did not follow project requirements as they are supposed to be followed, Kindly make sure that your project structure is following project requirements.
       - All the boilerplate that got created by the CRA (texts, images, and styles) ought to have gotten deleted.
       - There should be a `Calculator.js` file inside the components directory.
       - The `Calculator.js` component should be class-based.
       - The `App.js` component should be class-based.
       - The `App.js`(main component) should have the imported the `Calculator.js` component.
  - [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - [ ] Check the comments under the review.

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/App.js (hunk 1 above)
Comment (ReAliens) [reply src/App.js]: Okay, I will implement that but kindly does the name of components matter ??

**Review by bimbolabuari — APPROVED** [review]
# Project Approved 🚀

Hi @ReAliens 🥇,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ReAliens** [comment]
thanks a lot

**Pull request merged**